### PR TITLE
solve problems with invoking regular commands on process helper

### PR DESCRIPTION
**Pull request by cordoval**
Sent using [Gush](https://github.com/gushphp/gush)

### CODE REVIEW

**Comment by cordoval** [comment]
@romainneutron i am having some problems with permissions with process('rm -rf *.patch'); for instance in a /tmp/grace folder where the permissions of the files are cordova:wheel
by default the system declares these files belonging to wheel for some reason. When the process is run it does not give any output and just does not do anything which is very weird.

Is this symfony related?

File: src/Collabs/ZipperZippy.php [hunk 1]
```diff
@@ -25,12 +25,20 @@ public function zipAndBreak(array $patches)
...
-        $finder = (new Finder())
+        $files = (new Finder())
+            ->files()
+            ->in($cwd)
+            ->name('*.patch')
+        ;
+
+        $this->fs->remove($files);
+
+        $process = $this->helper->run('zip -v -s 2 compressAllFirst.zip --out splits.zip');
```
Comment (cordoval) [inline src/Collabs/ZipperZippy.php:37]: @sstok do you see why the zip -s 2 ... command fails here it is mute and does not say anything about what is wrong. stumped :flushed:

Comment (sstok) [inline src/Collabs/ZipperZippy.php:37]: Does the zip executable exists in with in the PATH? Try running it manually in the terminal.
What I was able to find on the zip command is that `-s` is used for the size and requires a unit like 2m as the default is bytes and a 2 byte archive is a bit (no pun) impossible :smile:

Comment (cordoval) [inline src/Collabs/ZipperZippy.php:37]: if i run manually it works, but it fails or says nothing of what is wrong when the process runs. Maybe i need to inherit path env vars? yeah the -s 2 works well in by hand. So it is 2MB by default i believe. Even if it does not weigh more than 2MB it still generates the file so clearly it is failing. Maybe it is related to the path? does it work on your end?

Comment (cordoval) [inline src/Collabs/ZipperZippy.php:37]: i believe it is PATH then? @sstok were you able to reproduce? i am on mac you are on windows though

**Pull request merged**